docs/logging-stdlib-usage.rst

```diff
@@ -0,0 +1,71 @@
+Integration with Python logging module
+--------------------------------------
+
+
+It's possible to tie the Python :mod:`logging` module directly into Google Cloud Logging. To use it,
+create a :class:`CloudLoggingHandler <google.cloud.logging.CloudLoggingHandler>` instance from your
```

Comment:
@dhermes This PR is about making snippets testable:  I don't want to hold it up to address issues about the stdlib handler usage docs.

If we want to open an issue for them, that is fine (we can link in @waprin, who wrote those).  I wanted them separate from the main usage docs so that we could get the snippets done for the "main" bits.